src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -356,8 +357,9 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx)
             maybePopulateDiagnosticEvents(cfg, out);
         }
     }
-    catch (std::exception&)
+    catch (std::exception& e)
     {
+        CLOG_DEBUG(Tx, "Exception caught while invoking host fn: {}", e.what());
```

Comment:
I missed in the earlier PR - Why does this no longer set the trapped result and return false? Was it removed so we could call `consumeRefundableSorobanResource`?

Reply:
Yeah, we need to do a bunch of stuff, such as record cpu/mem consumption for the sake of monitoring and record the refundable fee. Since `mSuccess` is false we'll set the proper error code in line 376.